template/template.go

```diff
@@ -238,6 +238,20 @@ func NewTemplateExpander(text string, name string, data interface{}, timestamp c
 				}
 				return fmt.Sprintf("%.4g%ss", v, prefix)
 			},
+			"humanizeTimestamp": func(v interface{}) string {
+				switch ts := v.(type) {
```

Comment:
As discussed on IRC, I changed this to just call `.Time()` on the `clientmodel.Timestamp` in the template, which outputs the right string representation automatically. So this function wasn't even needed anymore for my use case. Keeping it here (only supporting `float64` now) anyways, as it will be useful in the future.